Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift

```diff
@@ -177,6 +226,33 @@ final class CombineExtensionTests: XCTestCase {
             ]
             receive finished
             """)
+        
+        let recorder2 = StringRecorder()
+        
+        subscribeAndWait(
+            Fail<[Int], TestError>(error: TestError()).prettyPrint(to: recorder2).eraseToAnyPublisher()
+        )
+        
+        assertEqualLines(recorder2.contents.split(separator: "\n")[2...].joined(separator: "\n"),
+            """
+            receive failure:
+            TestError(
+                code: 1,
+                message: "This is the error"
+            )
+            """)
+    }
+    
+    private func subscribeAndWait(_ publisher: AnyPublisher<[Int], TestError>, sendHandler: (() -> Void)? = nil) {
```

Comment:
extracted common logics in each test cases.

Reply:
Great 👍